### DISCUSSION TITLE
Is this library production ready?

### DISCUSSION THREAD

**Discussion by fd6130** [czproject/git-php#66] [Q&A] [answered]
Hi, this library sounds great for me since i wanna try about managing git repository in php. But i saw there are lots of PR waiting to be merge or issue waiting to be solve..so i wonder is this library ready to use in production environment? 

Thanks.

**Comment by AlbertMN** [comment, marked as answer]
The PR's just seem to be improvements and extra features, so it depends whether you need any of those features, otherwise; yeah I'd say it's production ready if what the library currently offers covers what you require.
